app/src/main/AndroidManifest.xml

```diff
@@ -16,8 +16,9 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".MainActivity"></activity>
-        <activity android:name=".UpdatePasswordActivity"></activity>
+        <activity android:name=".MainActivity" />
+        <activity android:name=".UpdatePasswordActivity" />
```

Comment:
For cleanliness purposes (cleanliness of the PR, not the code), avoid doing this,,,, instead of that, if you want uniformity, change .PasswordForgottenActivity></activity>....
I will talk about PR cleanliness this today